README.md

```diff
@@ -5,9 +5,12 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/krassowski/jupyter-manim/master?filepath=Example.ipynb)
 [![Colab](https://camo.githubusercontent.com/52feade06f2fecbf006889a904d221e6a730c194/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667)](https://colab.research.google.com/gist/cjds/825d9acc80cb0b92bb877bbe2f468d70/example-jupyter-manim.ipynb)
 
-Integrates 3b1b's [manim](https://github.com/3b1b/manim) (animation engine for explanatory math videos)
+Integrates 3b1b's [ManimCairo](https://github.com/3b1b/manim) (cairo-backend branch)
 with Jupyter displaying the resulting video when using `%%manim` cell magic to wrap a scene definition.
 
+**WARNING**: This library only works for ManimCairo (the cairo-backend branch in 3b1b's version). It does not work for ManimCE (which already has Jupyter support by default)
```

Comment:
There is already a mention of that in the readme:

> Note: you do not need to install this extension when using the Manim Community edition as it already supports the %%manim magic

Reply:
Unfortunately, it seems like people still get it mixed up, including this [tutorial's creator](https://www.youtube.com/watch?v=2NLFpDDTS2U). I think it should be made as clear as possible to avoid confusion from new users.